libs/api/domains/auth/src/lib/dto/delegation.input.ts

```diff
@@ -0,0 +1,10 @@
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType('AuthDelegationInput')
+export class DelegationInput {
+  @Field((_) => String)
+  name!: string
+
+  @Field((_) => String)
+  toNationalId!: string
+}
```

Comment:
Do we want to create empty delegations? Could be a source of confusion.

Reply:
I would prefer to create the delegation after finishing the scope screen.

Reply:
Yeah, that was the thought. Create delegations with no scopes. Then work with that delegation to apply/remove scopes based on api-scopes... similar to editing a delegation, still just working with a single delegation.

"User created a delegation but didn't apply any scopes"

Could you further elaborate what possible confusions might occur?

Reply:
Just that the user may not be aware that something is saved after the "first step" of the wizard. Need to check if a delegation with no scopes would/should show up on the front page list and in login dropdowns?

If so, it could make some people surprised (thought they had cancelled when they saw the scope list), and others might think "that's it! you have access." even though they have no access.

Reply:
Yeah I see.
We previously talked about `delegation.isActive` which checks if there are any active scopes, so IMO we should not display inactive delegations in the dropdown